added a changelog for the pack readme validation (#1929)
demisto/demisto-sdk: CHANGELOG.md

* Added a validation to **validate**, checking that each script, integration and playbook have a README file. This validation only runs when the command is called with either the `-i` or the `-g` flag.
* Fixed a regression issue with **doc-review**, where  the `-g` flag did not work
* Improved the detection of errors in **doc-review** command.

> * Fixed an issue in **validate**, where README is not empty, is only checked in packs that contain playbooks, or written by partners. 

Reviewer: ```suggestion
* Fixed an issue in **validate**, where non-empty README files were only checked in packs that contain playbooks, or written by partners. 
```

Author: @dorschw don't think this is the best wording. please look again and confirm if ok by you.

> * When **validate** is validating *pack unique files* it will check for README existence only for packs that contain playbooks or writen by partners. 

Reviewer: ```suggestion
* Fixed an issue in **validate**, where README existence was only checked in packs that contain playbooks, or written by partners. 
```

Author: That is not really what i have done. the validation was once triggered for all packs containing use cases or written by partners. 
now it validates only for packs how contain playbooks or were written by partners. 
also made a better more clear error message. 
@dorschw

Reviewer: got it. plesae push the updated message.
